### PR TITLE
AG-343/AG-366 - Color and layout changes on gene network plot

### PR DESCRIPTION
**Pull request by mattcgenui**

### CODE REVIEW

**Comment by JessterB** [comment]
I pulled this down and ran it locally, and the colors are now updating properly! 

However, the sidebar content still doesn't update properly when you reselect the 'main' gene (but the link does go to the 'main' gene's results).

**Comment by JessterB** [comment]
I pulled this down and ran it locally, and the colors are now updating properly! 

However, the sidebar content still doesn't update properly when you reselect the 'main' gene (but the link does go to the 'main' gene's results).

File: src/app/genes/gene-overview/evidence-menu/rna/gene-network/gene-network.component.ts [hunk 1]
```diff
@@ -156,30 +156,28 @@ export class GeneNetworkComponent implements OnInit {
 
     updategene(event: Gene) {
         const self = this;
-        if (event.ensembl_gene_id !== this.geneService.getCurrentGene().ensembl_gene_id) {
```
Comment (jay-hodgson) [inline src/app/genes/gene-overview/evidence-menu/rna/gene-network/gene-network.component.ts:159]: Did this used to ignore a no-op?

Comment (mattcgenui) [reply src/app/genes/gene-overview/evidence-menu/rna/gene-network/gene-network.component.ts:159]: Yes, if you click a different gene in the network and click back on the current gene, the sidebar data does not update. There is probably room for improvement by caching the data and avoid reloading at every click

**Review by jay-hodgson — APPROVED** [review]

**Pull request merged**